README.md

```diff
@@ -102,10 +111,63 @@ emitter.unbind()
 emitter.bind(10000) // Samples will now fire once every 10 seconds.
 ```
 
-## License
+## Testing
+
+This module includes a list of unit and functional tests.  To run these tests, use the following command
+
+    $ npm run test
+
+You may also run individual test suites with the following commands
+
+    $ npm run unit
+    $ npm run integration
```

Comment:
There are native tests that didn't appear to be ran by CI. They *might* work via commenting some build code but that didn't seem to work quite right. So for now, excluding from here. I'm not sure if these were deprecated and never removed (prob covered via integration tests), setup is just poorly documented, or what but also don't want to bog down too hard on it.